modules/storages/app/common/storages/peripherals/storage_interaction/nextcloud/copy_template_folder_command.rb

```diff
@@ -33,28 +33,27 @@ module Nextcloud
         class CopyTemplateFolderCommand
           using ServiceResultRefinements
 
-          def self.call(storage:, source_path:, destination_path:)
-            new(storage).call(source_path:, destination_path:)
+          def self.call(auth_strategy:, storage:, source_path:, destination_path:)
```

Comment:
🟢 good thing I saw that now, was going to start with that command almost myself.